pycode/examples/secir_groups.py

```diff
@@ -52,55 +54,54 @@ def run_secir_groups_simulation():
+        model.populations[AgeGroup(i), Index_InfectionState(State.Exposed)] = 100
+        model.populations[AgeGroup(i), Index_InfectionState(State.Carrier)] = 50
+        model.populations[AgeGroup(i), Index_InfectionState(State.Infected)] = 50
+        model.populations[AgeGroup(i), Index_InfectionState(State.Hospitalized)] = 20
+        model.populations[AgeGroup(i), Index_InfectionState(State.ICU)] = 10
+        model.populations[AgeGroup(i), Index_InfectionState(State.Recovered)] = 10
+        model.populations[AgeGroup(i), Index_InfectionState(State.Dead)] = 0
+        model.populations.set_difference_from_total((AgeGroup(i), Index_InfectionState(State.Susceptible)), populations[i])
 
         # Compartment transition propabilities
-        model.parameters.probabilities[i].set_infection_from_contact(1.0)
-        model.parameters.probabilities[i].set_carrier_infectability(0.67)
-        model.parameters.probabilities[i].set_asymp_per_infectious(0.09)
-        model.parameters.probabilities[i].set_risk_from_symptomatic(0.25)
-        model.parameters.probabilities[i].set_hospitalized_per_infectious(0.2)
-        model.parameters.probabilities[i].set_icu_per_hospitalized(0.25)
-        model.parameters.probabilities[i].set_dead_per_icu(0.3)
-        
+           
+        model.parameters.RelativeCarrierInfectability[AgeGroup(i)] = 0.67  #TODO: must this parameter be set? 
+        model.parameters.InfectionProbabilityFromContact[AgeGroup(i)] = 1.0
+        model.parameters.AsymptoticCasesPerInfectious[AgeGroup(i)] = 0.09  # 0.01-0.16
+        model.parameters.RiskOfInfectionFromSympomatic[AgeGroup(i)] = 0.25  # 0.05-0.5
+        model.parameters.HospitalizedCasesPerInfectious[AgeGroup(i)] = 0.2  # 0.1-0.35
+        model.parameters.ICUCasesPerHospitalized[AgeGroup(i)] = 0.25  # 0.15-0.4
+        model.parameters.DeathsPerHospitalized[AgeGroup(i)] = 0.3  # 0.15-0.77
+        #TODO what about MaxRiskOfInfectionFromSympomatic?
     
     # set contact rates and emulate some mitigations
     # set contact frequency matrix
-    model.parameters.get_contact_patterns().cont_freq_mat[0].baseline = np.loadtxt(baseline_contact_matrix0) \
+    model.parameters.ContactPatterns.cont_freq_mat[0].baseline = np.loadtxt(baseline_contact_matrix0) \
         + np.loadtxt(baseline_contact_matrix1) + np.loadtxt(baseline_contact_matrix2) + np.loadtxt(baseline_contact_matrix3)
-    model.parameters.get_contact_patterns().cont_freq_mat[0].minimum = np.ones((num_groups, num_groups)) * 0
-
-    # Define Damping on Contacts
-    model.parameters.get_contact_patterns().cont_freq_mat.add_damping(secir.Damping(np.ones((num_groups, num_groups)) * 0.9, 30, 0, 0))
+    model.parameters.ContactPatterns.cont_freq_mat[0].minimum = np.ones((num_groups, num_groups)) * 0
+    model.parameters.ContactPatterns.cont_freq_mat.add_damping(Damping(coeffs = np.ones((num_groups, num_groups)) * 0.7, t = 30.0, level = 0, type = 0))
```

Comment:
Please stick to the same values in the examples (0.9 before, 0.7 now) if there isn't any reason to change.

Reply:
I used the values that are in the testing example assuming it should be equal everywhere.